src/js/jsx/Scrim.jsx

```diff
@@ -246,7 +248,8 @@ define(function (require, exports, module) {
                 transform = this.state.transform,
                 overlays = !disabled && this.state.overlaysEnabled,
                 transformString = this._getTransformString(transform),
-                toolOverlay = (overlays && transform) ? this._renderToolOverlay(transformString) : null;
+                toolOverlay = (overlays && transform) ? this._renderToolOverlay(transformString) : null,
+                policyOverlay = false ? (<PolicyOverlay/>) : null;
```

Comment:
Because I had to remove debug.json... Once I find a nicer solution, I'll switch to that.